include/dlaf/eigensolver/gen_eigensolver.h

```diff
@@ -108,14 +252,16 @@ void hermitian_generalized_eigensolver(blas::Uplo uplo, Matrix<T, D>& mat_a, Mat
 /// @pre @p mat_a has blocksize (NB x NB)
 /// @pre @p mat_a has tilesize (NB x NB)
 ///
-/// @param mat_b contains the Hermitian positive definite matrix B
+/// @param mat_b contains the Cholesky factorisation of the Hermitian positive definite matrix B
```

Comment:
Please add this comment here as well...
https://github.com/eth-cscs/DLA-Future/blob/master/include/dlaf/eigensolver/gen_to_std.h#L43